### PR TITLE
Update tuples.md

### PR DESCRIPTION
**Pull request by vnextcoder**
Fix Lifted Conversions Link

## Summary

Describe your changes here.  Fix the Lifted Conversions link

Fixes #Issue_Number (if available) - none

### CODE REVIEW

File: docs/csharp/tuples.md [hunk 1]
```diff
@@ -153,7 +153,8 @@ Beginning with C# 7.3, tuple types support the `==` and `!=` operators. These op
 
 [!code-csharp[TupleEquality](../../samples/snippets/csharp/tuples/tuples/program.cs#Equality "Testing tuples for equality")]
 
-There are several rules that make tuple equality tests more convenient. Tuple equality performs [lifted conversions](/dotnet/csharp/language-reference/language-specification/conversions.md#lifted-conversion-operators) if one of the tuples is a nullable tuple, as shown in the following code:
+There are several rules that make tuple equality tests more convenient. Tuple equality performs [lifted conversions](/dotnet/csharp/language-reference/language-specification/conversions#lifted-conversion-operators) if one of the tuples is a nullable tuple, as shown in the following code:
```
Comment (mairaw) [inline docs/csharp/tuples.md]: That does fix the broken link but would you be so kind to change this to a relative url so the link gets validated during our build process?

/dotnet/csharp/language-reference/language-specification/conversions#lifted-conversion-operators -> ./language-reference/language-specification/conversions.md#lifted-conversion-operators

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you @vnextcoder! Your PR fixes the issue but the link wasn't following our guidelines for internal links. Can you make the change I suggested?

**Review by mairaw — APPROVED** [review]
Ah, my apologies. Your fix was correct. I forgot that link to the content outside our repo. Thank you @vnextcoder!

**Comment by vnextcoder** [comment]
Sorry @mairaw , I couldn't see your messages earlier. I hope all fine with the PR and change I suggested.

**Pull request merged**